src/integrationTest/java/uk/gov/hmcts/divorce/document/content/NoticeOfProceedingJointJudicialSeparationContentIT.java

```diff
@@ -0,0 +1,112 @@
+import static uk.gov.hmcts.divorce.document.content.NoticeOfProceedingJointJudicialSeparationContent.JUDICIAL_SEPARATION_PROCEEDINGS;
+import static uk.gov.hmcts.divorce.document.content.NoticeOfProceedingJointJudicialSeparationContent.MARRIED_TO_MORE_THAN_ONE_PERSON;
+import static uk.gov.hmcts.divorce.document.content.NoticeOfProceedingJointJudicialSeparationContent.REISSUED_DATE;
+import static uk.gov.hmcts.divorce.notification.CommonContent.ADDRESS;
+import static uk.gov.hmcts.divorce.notification.FormatUtil.formatId;
+import static uk.gov.hmcts.divorce.testutil.TestConstants.TEST_CASE_ID;
+import static uk.gov.hmcts.divorce.testutil.TestConstants.TEST_FIRST_NAME;
+import static uk.gov.hmcts.divorce.testutil.TestConstants.TEST_LAST_NAME;
+import static uk.gov.hmcts.divorce.testutil.TestDataHelper.caseData;
+
+@ExtendWith(SpringExtension.class)
+@SpringBootTest
+public class NoticeOfProceedingJointJudicialSeparationContentIT {
+
+    @Autowired
+    private CommonContent commonContent;
+
+    @Autowired
+    private DocmosisCommonContent docmosisCommonContent;
+
+    @Autowired
+    private NoticeOfProceedingJointJudicialSeparationContent noticeOfProceedingContentJudicialSeparation;
+
+    public static final String MARRIED_TO_MORE_THAN_ONE_PERSON_TEXT = "You must tell the court if you’ve been married to more than"
+        + " one person during this marriage.";
+
+    @Test
+    public void shouldSuccessfullyApplyContentFromCaseDataForGeneratingNoticeOfProceedingJointDocForJudicialSeparation() {
+        CaseData caseData = caseData();
+        caseData.setDivorceOrDissolution(DivorceOrDissolution.DIVORCE);
+        caseData.getApplicant1().setFirstName(TEST_FIRST_NAME);
+        caseData.getApplicant1().setLastName(TEST_LAST_NAME);
+        caseData.getApplicant1().setGender(MALE);
+        caseData.getApplicant1().setAddress(
+            AddressGlobalUK.builder()
+                .addressLine1("line 1")
+                .postTown("town")
+                .postCode("postcode")
+                .country("UK")
+                .build()
+        );
+        caseData.getApplicant2().setGender(FEMALE);
+
+        caseData.getApplication().setIssueDate(LocalDate.of(2021, 6, 18));
+        caseData.getApplication().setReissueDate(LocalDate.of(2021, 7, 18));
+        caseData.setDueDate(LocalDate.of(2021, 6, 19));
+
+        var ctscContactDetails = CtscContactDetails
+            .builder()
+            .poBox("PO Box 13226")
+            .postcode("CM20 9UG")
+            .town("Harlow")
+            .build();
+
+        Map<String, Object> expectedEntries = new LinkedHashMap<>();
+        expectedEntries.put(CASE_REFERENCE, formatId(1616591401473378L));
+        expectedEntries.put(FIRST_NAME, TEST_FIRST_NAME);
+        expectedEntries.put(LAST_NAME, TEST_LAST_NAME);
+        expectedEntries.put(ISSUE_DATE, "18 June 2021");
+        expectedEntries.put(REISSUED_DATE, "Reissued on: 18 July 2021");
```

Comment:
we don't need Reissued on: ?

Reply:
For my template, the logic is done in NoticeOfProceedingJointJudicialSeparationContent.java. So if there's a reissue date it will pass in the text and the date together